nalu.C

```diff
@@ -25,6 +25,7 @@
 // input params
 #include <stk_util/environment/OptionsSpecification.hpp> 
 #include <stk_util/environment/ParseCommandLineArgs.hpp> 
+#include <stk_util/environment/ParsedOptions.hpp>
```

Comment:
Why was this include added?

Reply:
We found out that building Nalu with the latest Trilinos build (with latest snapshot of STK) causes incomplete type compile error on `ParsedOption` vars.
It seemed that `ParsedOptions.hpp` was previously included by `ParseCommandLineArgs.hpp` but was removed recently.

Reply:
ok, thanks for the explanation.